updates/cesium/Update2_Sona

```diff
@@ -0,0 +1,8 @@
+Past week: 
+> Alejandro and I updated the read me file and made it more detailed. Alejandro made the pull request.
+
+Next week:
+> Will work on one of the tasks. I will also read the source code to learn more about the project.
+
+Issues:
+> We could not find the license information.
```

Comment:
Please use the license file from Cesium: https://github.com/cesium-ml/cesium/blob/master/LICENSE.txt